### PR TITLE
Add more help text

### PR DESCRIPTION
**Pull request by mfairchild365**
This includes help text for several more common errors.

* obsolete valign, align, and width attributes
* obsolete font element
* missing alt attributes
* unencoded ampersands

### CODE REVIEW

File: help_text/fa13d9f7d625dfa5758b294ad1d308da.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+This error means that there is an image (`img`) element on the page without an `alt` attribute.  An `alt` attribute should contain alternative for the image in cases where the image can not be presented.  Some examples of these cases are when the image does not load or when the page is read with a screen reader for the visually impaired.
```
Comment (kabel) [inline help_text/fa13d9f7d625dfa5758b294ad1d308da.md]: Missing word after "alternative", or did you mean for it to be a noun.

File: help_text/4c1560dca6a927e575ca4fc7f77516ad.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+This error means that there are unencoded ampersands (&) in your markup.  HTML uses the ampersand character to start a character reference, and is thus a reserved symbol.  Instead of typing `&`, type the character reference `&amp;` instead.
+
+The following example is **not valid** because it does not use a character reference:
+
+```
+today & tomorrow
```
Comment (kabel) [inline help_text/4c1560dca6a927e575ca4fc7f77516ad.md]: Technically speaking this example _is_ valid, but will create a notice that you should use the full character reference. The error occurs when the ampersand is used in a place that it is ambiguous whether is it a literal or the delimeter (when there is no space before/after).

Comment (mfairchild365) [inline help_text/4c1560dca6a927e575ca4fc7f77516ad.md]: fixed in https://github.com/mfairchild365/metric_w3c_html/commit/aeedcc35ee0bbac79d8a58a98bd7fc517cb5050d

File: help_text/fa13d9f7d625dfa5758b294ad1d308da.md (hunk 1 above)
Comment (mfairchild365) [inline help_text/fa13d9f7d625dfa5758b294ad1d308da.md]: fixed in https://github.com/mfairchild365/metric_w3c_html/commit/4da5d59c1c036975f5209f4c63f80a14afe9950e

**Pull request merged**